### ISSUE TITLE
Safari not support AVIF 

### ISSUE THREAD

**Issue by alexvamp** [eworm-de/routeros-scripts#20]
Safari not support AVIF 

https://avif.io/blog/tutorials/safari/#safariavifsupportsoon

**Comment by eworm-de** [comment]
Yeah, Apple does not have AVIF support in place in its products. Same with Microsoft and its Edge browser.

Tested with everything else and works just fine:

* [Firefox](https://www.mozilla.org/firefox/browsers/)
* [Chrome](https://www.google.de/chrome/)
* [Opera](https://www.opera.com/)
* [Vivaldi](https://vivaldi.com/)

**Comment by xlionjuan** [comment]
The domain avif .io is now redirected to gambling website

**Comment by eworm-de** [comment]
So what? That's not my domain, and I do not link there.
